apps/prairielearn/src/lib/groups.sql

```diff
@@ -85,12 +85,17 @@ SELECT
 FROM
   create_group AS cg;
 
--- BLOCK get_group_members
+-- BLOCK select_group
+SELECT
+  *
+FROM
+  groups
+WHERE
+  id = $group_id;
+
+-- BLOCK select_group_members
 SELECT DISTINCT
```

Comment:
I'm not 100% sure why this was `SELECT DISTINCT`. This might be unneeded now?

Reply:
Indeed, I think it's unnecessary. Also the JOIN with groups is not needed in this query.

Reply:
Good call! I simplified the query in 06c2044, can you check my work?

Reply:
You kept the Distinct?

Reply:
🤦 I got so preoccupied with removing the join...

Reply:
Removed in 2e63de0.